lib/common.ex

```diff
@@ -69,14 +69,22 @@ defmodule ScenicNew.Common do
     end
   end
 
-  def check_mod_name_validity!(name) do
+  def check_mod_name_validity!(name) when is_binary(name) do
     unless name =~ Regex.recompile!(~r/^[A-Z]\w*(\.[A-Z]\w*)*$/) do
       Mix.raise(
         "Module name must be a valid Elixir alias (for example: Foo.Bar), got: #{inspect(name)}"
       )
     end
   end
 
+  def check_mod_name_validity!(name) when is_atom(name) do
+    unless Atom.to_string(name) =~ Regex.recompile!(~r/^[A-Z]\w*(\.[A-Z]\w*)*$/) do
+      Mix.raise(
+        "Module name must be a valid Elixir alias (for example: Foo.Bar), got: #{inspect(name)}"
+      )
+    end
+  end
```

Comment:
This should not be needed, see:
https://github.com/phoenixframework/phoenix/blob/v1.5.7/lib/mix/tasks/phx.gen.presence.ex#L1